src/Features/Core/Portable/AddImport/CodeActions/ParentInstallPackageCodeAction.cs

```diff
@@ -35,51 +33,60 @@ private class ParentCodeAction : CodeAction.CodeActionWithNestedActions
                 /// navigate through, and we don't want our child items confusingly being added to the
                 /// top level light-bulb where it's not clear what effect they would have if invoked.
                 /// </summary>
-                public ParentCodeAction(
-                    PackageReference reference,
+                public ParentInstallPackageCodeAction(
+                    IPackageInstallerService installerService,
```

Comment:
We do need to pass along the installer service.  After all, we need to actually install the package somehow.  But hte code action is otherwise divorced from any other objects that could perform logic.